cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHttpService.java

```diff
@@ -63,6 +63,7 @@ public StreamHttpService(CConfiguration cConf, DiscoveryService discoveryService
       .setHandlerHooks(ImmutableList.of(new MetricsReporterHook(metricsCollectionService,
                                                                 Constants.Stream.STREAM_HANDLER)))
       .setHost(cConf.get(Constants.Stream.ADDRESS))
+      .setPort(cConf.getInt(Constants.Stream.PORT))
```

Comment:
The stream service does not run in the master node, but in a Yarn container. I am not sure whether it is a good idea to hardcode a port for services that run in containers, because there is no guarantee that the port is not taken by a service that runs in a different container, so this would randomly introduce failures at startup.